src/data/links.json

```diff
@@ -55,7 +55,8 @@
         {
           "name": "C API Summit",
           "path": "/capi"
-        }
+        },
+        { "name": "Humble Data", "path": "/humble-data" }
```

Comment:
```suggestion
        {
          "name": "Humble Data",
          "path": "/humble-data"
        }
```

Reply:
hmm, this was done by precommit on my system?

Reply:
Oh really? Well, we better keep it that way then, or the bot will bug us until eternity :D